src/pages/account.tsx

```diff
@@ -100,47 +92,41 @@ const Activity: React.FunctionComponent<ActivityProps> = ({ transactions }) => {
 const SessionComponent = () => {
   const { data: session, status } = useSession();
 
-  const user = api.users.findUserByEmail.useQuery({
-    email: session?.user.email ?? "0",
-  }).data;
-  const transactions = api.users.getUserTransactionsById.useQuery({
-    id: user?.id ?? "0",
-  }).data;
+  const user = api.users.getUserBySession.useQuery();
 
   if (session) {
     return (
       <>
-        <div className="flex flex-col gap-4 rounded bg-white p-4 text-lg font-medium text-[#323232] sm:w-1/3">
-          <h3 className="text-xl font-semibold">General Information</h3>
-          <section className="flex flex-col gap-2">
-            <div className="flex gap-2.5">
-              <p className="font-medium">Full name:</p>
-              <p className="font-light">{session?.user.name}</p>
-            </div>
-            {session?.user.email && (
+        {user.isLoading ? (
+          <CircularProgress />
+        ) : (
+          <div className="flex flex-col gap-4 rounded bg-white p-4 text-lg font-medium text-[#323232] sm:w-1/3">
+            <h3 className="text-xl font-semibold">General Information</h3>
+
+            <section className="flex flex-col gap-2">
               <div className="flex gap-2.5">
-                <p className="font-medium">Company:</p>
-                <p className="font-light">{session.user.email.split("@")[1]}</p>
+                <p className="font-medium">Full name:</p>
+                <p className="font-light">{user.data?.name}</p>
               </div>
-            )}
-          </section>
-          <hr />
-          <h3 className="text-xl font-semibold">Contact Information</h3>
-          <section className="flex flex-col gap-1">
-            <div className="flex gap-2.5">
-              <p className="font-medium">Email:</p>
-              <p className="font-light">{session?.user.email}</p>
-            </div>
-            {/* {session?.user.number && ( // TODO: there is no phone number in the session
+              {user.data?.email && (
+                <div className="flex gap-2.5">
```

Comment:
Un nice to have es usar componentes de MaterialUI. Por ejemplo en vez de <div> usar <Grid> de material UI: https://mui.com/material-ui/react-grid/

Con eso, podemos estar 100% que la pagina se va a adaptar a cualquier resolucion de pantalla.